ext/gd/tests/bug43475.phpt

```diff
@@ -7,6 +7,9 @@ gd
     if (!GD_BUNDLED && version_compare(GD_VERSION, '2.2.2', '<')) {
         die("skip test requires GD 2.2.2 or higher");
     }
+    if (!(imagetypes() & IMG_PNG)) {
+        die("skip No PNG support");
+    }
```

Comment:
Does this really require PNG? Or can this test not also use different formats?

Reply:
It  _could_, but right now it's doing `$ex = imagecreatefrompng(__DIR__ . '/bug43475.png');` which requires PNG support.

Reply:
Yeah that's fair, I realized afterwards that a lot of the tests also need it just to check that it is actually creating the image correctly